testdata/xml-data/test-data-systematic.xml

```diff
@@ -84,8 +84,9 @@
             <text encoding="xml">&amp;</text>
             <text encoding="xml"><em>text</em></text>
             <text encoding="xml">text &lt;not a tag&gt; text</text>
-            <text permissions="prop-default" encoding="xml">
-                This is <em>bold and <strong>strong</strong></em> text! It contains links to all resources:
+            <text encoding="xml">This is <em>italicized and <strong>bold</strong></em> text! </text>
+            <text encoding="xml" permissions="prop-default">
+                This text contains links to all resources:
                 <a class="salsah-link" href="IRI:test_thing_0:IRI">test_thing_0</a>
```

Comment:
This is the change that I mentioned above: I splitted up 1 text value into 2 values.